### DISCUSSION TITLE
AXAML Hot Reload support ?

### DISCUSSION THREAD

**Discussion by xwang-888** [AvaloniaUI/Avalonia#12639] [Ideas]
AXAML Hot Reload support ?

**Comment by thevortexcloud** [comment]
There is an official project for it. I have never tried it so I don't know how well it works.


https://github.com/AvaloniaUI/Live.Avalonia

**Reply by kyuranger** [reply]
This package has not been updated for a long time. I have tested it in 11.0.0 and it doesn't work any.

**Comment by timunie** [comment]
Duplicate of #3266

**Comment by 996666925** [comment]
Go try [Avalonia.css](https://github.com/liwuqingxin/Avalonia.Css), which can perform style hot reloading
![acss](https://github.com/AvaloniaUI/Avalonia/assets/52450772/70e36184-8e27-43ab-95aa-206d66762333)

**Reply by xwang-888** [reply]
Thanks, I want something like WPF, instead of writing acss.
